src/components/app/App.tsx

```diff
@@ -84,6 +87,20 @@ export class Evergreen extends React.Component<Props, State> {
       APIClient: rest.EvergreenClient(configObj.user, configObj.api_key, configObj.api_url, configObj.ui_url),
     });
   }
+
+  private tryLoadConfig = () => {
+    fetch(configPath).then((resp: Response) => {
```

Comment:
nit: dunno if we have a convention or care, but there's inconsistent spacing after the parens after the `then`